src/main/java/fun/epak/pak/controller/ProfileController.java

```diff
@@ -47,9 +49,23 @@ public String getOtherProfile(@PathVariable long id, Model model, Principal prin
         return "/profile/otherProfile";
     }
 
-    @PostMapping("/subscriptions")
-    public RedirectView changeSubscriptionList(long id, Principal principal) {
-        userService.changeUserSubscriptionList(principal.getName(), id);
-        return new RedirectView("/profile/" + id);
+    @PostMapping("/subscribe")
+    public RedirectView subscribeUser(long subscribedUserId, Principal principal) {
+        userService.subscribeUser(principal.getName(), subscribedUserId);
```

Comment:
Można by było wydzielić wspólną metodę dla tych endpointów, bo wyglądają praktycznie tak samo. To którą metodę z serwisu wołamy można by było przekazać jako `BiFunction`